### PR TITLE
Seed Project Path Update

### PR DESCRIPTION
**Pull request by mehreencs87**

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

File: articles/server-apis/ruby/authenticate.md [hunk 1]
```diff
@@ -13,7 +13,13 @@ This tutorial and seed project have been tested with the following:
 * Ruby 2.1.7
 :::
 
-<%= include('../../_includes/_package', { pkgRepo: 'ruby-auth0', pkgBranch: 'master', pkgPath: 'examples/ruby-api', pkgFilePath: null, pkgType: 'server' }) %>
+<%= include('../../_includes/_package', {
+pkgRepo: 'auth0-rubyonrails-sample',
```
Comment (mpaktiti) [inline articles/server-apis/ruby/authenticate.md:18]: @mehreencs87 Hi, another change was introduced and we need to apply it before this PR is merged. `pkgOrg` was added:
- if the sample is under `auth0-samples` repo add the line: `pkgOrg: 'auth0-samples'`
- if the sample is under `auth0` repo add the line: `pkgOrg: 'auth0'`
  Details: https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#seed-projects

Can you please update this? Thanks.

**Comment by mehreencs87** [comment]
@mpaktiti check the latest commit.

File: articles/server-apis/ruby/authenticate.md [hunk 2]
```diff
@@ -13,7 +13,14 @@ This tutorial and seed project have been tested with the following:
 * Ruby 2.1.7
 :::
 
-<%= include('../../_includes/_package', { pkgRepo: 'ruby-auth0', pkgBranch: 'master', pkgPath: 'examples/ruby-api', pkgFilePath: null, pkgType: 'server' }) %>
+<%= include('../../_includes/_package', {
+pkgOrg: 'auth0-samples',
+pkgRepo: 'auth0-rubyonrails-sample', 
+githubUrl: 'https://github.com/auth0-samples/auth0-rubyonrails-sample/tree/master/00-Starter-Seed',
+pkgBranch: 'master', 
+pkgPath: '00-Starter-Seed/ruby-api',
```
Comment (mpaktiti) [inline articles/server-apis/ruby/authenticate.md]: @mehreencs87 The [download URL](https://auth0.com/docs/package/auth0-rubyonrails-sample/master?org=auth0-samples&path=00-Starter-Seed/ruby-api&file_path=&type=server) returns an error.
There is no `ruby-api` folder under https://github.com/auth0-samples/auth0-rubyonrails-sample/tree/master/00-Starter-Seed`. I removed it locally and the link worked. Can you update this please?

**Comment by mehreencs87** [comment]
@mpaktiti check the latest commit.

**Pull request merged**